src/counts.jl

```diff
@@ -238,9 +241,53 @@ If a weighting vector `wv` is specified, the sum of the weights is used rather t
 raw counts.
 """
 function addcounts!(cm::Dict{T}, x::AbstractArray{T}) where T
+    # if it's of bits type then can speed things up using radix sort
+    # from heuristics it was found that 2^16 = 65536 is the point at which
+    # radixsort is more performant
+    if radixsort_safe(T) & length(x) > 65536
+        addcounts_radixsort!(cm, x)
+        return cm
+    else
+        addcounts_dict!(cm,x)
+        return cm
+    end
+end
+
+function addcounts_dict!(cm::Dict{T}, x::AbstractArray{T}) where T
     for v in x
-        cm[v] = get(cm, v, 0) + 1
+        index = ht_keyindex2(cm, v)
+        if index > 0
+          @inbounds cm.keys[index] = v
```

Comment:
I don't think that's needed, is it? At least FreqTables doesn't use it and it works.

Also, use four spaces for indentation.

Reply:
I removed the cm.keys[index] = v that seems redudant.